BatteriesDocs/lean-toolchain

```diff
@@ -0,0 +1 @@
+../lean-toolchain
```

Comment:
I did not know you could do this. But I think it's not necessary, elan should pick the lean-toolchain from the root if you don't supply one here.

Reply:
I don't know how portable github symliks are so that may well be a better idea!